README.md

```diff
@@ -209,6 +209,24 @@ try {
 }
 ```
 
+### Renew Authentication
+
+Creates a new request to renew the authentication and get fresh new credentials using a valid Refresh Token.
```

Comment:
do we need to specify like in `### Request Token for Audience - /oauth/token` that this is the "new" /oauth/token refresh token?